### ISSUE TITLE
Grass causes a glitch when terrain and water are hidden.

### ISSUE THREAD

**Issue by Retrogamebloke** [TheGameCreators/GameGuruRepo#821]
I have noticed that when you hide everything and have grass in your level, you see a slight white/grey effect where the terrain should be. I include an image to attempt to show it. You can clearly see the black box around my logo object, which was practically invisible before (just blanking the odd star).

I have narrowed it down to the grass. Put in grass and the effect is visible in the test level as well.

I am currently still using the May version due to speed issues with the latest build, so can't say for sure if it is in the new version or not.

![backdropglitch](https://user-images.githubusercontent.com/44620846/91596150-a9028880-e95c-11ea-9847-246572648245.jpg)

**Comment by plemsoft** [comment]
@Retrogamebloke Thanks for reporting this issue,

I been staring at the image and cant see any "slight white/grey effect" ? could you circle it in your image. 
If you mean the fading from black to "blueish" in the background , then its the skybox that have this.

Anyway let me know :)

**Comment by plemsoft** [comment]
@Retrogamebloke I did not hear back from you , was it the skybox ?

**Comment by Retrogamebloke** [comment]
Ah missed the mail. Been awhile since I worked on this now but here is an old video showing the white glow.

https://user-images.githubusercontent.com/44620846/109971301-000f5e80-7cee-11eb-889d-f2133aaeddde.mp4
This second vid is what it looked like on a separate map (same skybox)
https://user-images.githubusercontent.com/44620846/109971906-a22f4680-7cee-11eb-9b74-dd1262ea1464.mp4

Seem to remember it vanishes in F11 mode when you turned off grass.

**Comment by plemsoft** [comment]
@Retrogamebloke Thanks for getting back , when i look at the second video i can also see the blue shine , i think it depends on what direction your pointing the camera at , but it looks like the skybox, could you try to edit the skybox textures and make it completely black , i think that will solve the issue :)


![blueish](https://user-images.githubusercontent.com/18662417/109984751-6010ff80-7d04-11eb-9243-241dfcd23747.png)

I that dont help, could you make a small .fpm i can test and see the issue ? , you can e-mail a download link to me at: plemsoft@plemsoft.com

Thanks for your help.

**Comment by Retrogamebloke** [comment]
It's been an age since I worked on this little demo. I still have the map and can make a standalone of it if you want, but I'm not really going to look into editing skyboxes just for this issue, as I am 100% sure it is _not_ the skybox and will be a waste of time. If it were, it would show in the first demo which uses exactly the same one. There's no logic to that; both demos point in the same direction. I made a map with just the intro, which worked fine (hidden terrain, but nothing on it) and a map that had the intro and also full map hidden, ready to be shown after the intro. I got the white blur with the second map, the first, which was identical bar having anything on the map, worked fine. It vanished if I hid the grass using F11 but that is not an option for any game release.
I can send a standalone of the demo, but an fpm is useless as it is ALL custom media in the main. I haven't run it with the latest version, but if you haven't found the issue, then I imagine it is still there. It's a rarity I am sure, but does exist :)
